### DISCUSSION TITLE
Seeking Laser Engraving Setup Assistance

### DISCUSSION THREAD

**Discussion by coyola21** [arkypita/LaserGRBL#2185] [Q&A]
Hi there,

I'm brand new to laser engraving and, due to dyslexia and a traumatic brain injury, navigating beyond basic instructions is challenging. I've done some test runs but need detailed step-by-step guidance for configuring my laser. Are there comprehensive setup guides available, or does anyone with Zoom mind helping me remotely? I'm a disabled female veteran starting a new venture to empower individuals on a recovery journey. I aim to find a new hobby and give back. Any help is appreciated.

Thank you,
Christina

**Comment by Tribo-2dot0** [comment]
Não domino outro idioma além do português, talvez alguem lhe ajude com acesso remoto, mas podes começar lendo atentamente aqui:
http://lasergrbl.com/usage/
Leia também a página FAQ.

**Comment by coyola21** [comment]
Thank you

Christina Oyola,
 Founder and CEOThat Day Company & HerStory UnveiledNAWBO - Greater Philadelphia Chapter
Cell: 856-938-8084Work: 267-738-7564
 Bensalem, Pennsylvania 
https://www.paypal.com/donate/?hosted_button_id=2STSC7B38MC44

Follow me on all platformshttps://linktr.ee/TheChristinaeffect?utm_source=linktree_admin_share
 

    On Wednesday, January 24, 2024 at 09:06:14 PM EST, Tribo-2dot0 ***@***.***> wrote:  
 
 


Não domino outro idioma além do português, talvez alguem lhe ajude com acesso remoto, mas podes começar lendo atentamente aqui:
http://lasergrbl.com/usage/
Leia também a página FAQ.
